classes/ClaytonTestClass1.cls

```diff
@@ -0,0 +1,953 @@
+public class ClaytonTestClass1 {
+    public ClaytonTestClass1(){
+        List<Case> cList = [Select Id,CaseNumber, AccountId, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. I suggest to introduce a `WHERE` clause or cap the max number of records returned adding a `LIMIT` clause to the query.